cmd/osdsapiserver/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Docker build usage:
-# 	docker build . -t opensdsio/opensds-apiserver:latest
+# 	docker build . -t sodafoundation/apiserver:latest
 # Docker run usage:
-# 	docker run -d --net=host -v /etc/opensds:/etc/opensds opensdsio/opensds-apiserver:latest
+# 	docker run -d --net=host -v /etc/opensds:/etc/opensds sodafoundation/apiserver:latest
 
 FROM ubuntu:16.04
 MAINTAINER Leon Wang <wanghui71leon@gmail.com>
```

Comment:
This mandatory? Else, can we remove and let all come to github of the project and its maintainer's list?
If mandatory, we can give like SODA Foundation <support@sodafoundation.io>